bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/meeting/dto/MeetingTeamLocationResponse.kt

```diff
@@ -0,0 +1,24 @@
+package com.studentcenter.weave.bootstrap.meeting.dto
+
+import com.studentcenter.weave.domain.meeting.enums.Location
+
+data class MeetingTeamLocationResponse(
+    val locations: List<MeetingTeamLocationDto>
+) {
+
+    data class MeetingTeamLocationDto(
+        val item: String,
+        val value: String
```

Comment:
p2) 클라이언트가 필요한 정보가 내려가야할 것 같아요. 저희가 어떤 이름으로 관리하는지는 중요하지 않은 정보 같습니다.
```suggestion
        val name: String,
        val isisCapitalArea: Boolean
```

Reply:
![image](https://github.com/Student-Center/weave-server/assets/59856002/30553b84-a80f-4810-9a73-5848179c6f39)
요부분 클라분들이 미팅 팀 생성 시에 저희가 관리하는 ENUM의 item명으로 입력을 하고 계셔서, 해당 정보 포함해서 응답 api 수정했습니다!